py/torch_tensorrt/dynamo/conversion/impl/shuffle.py

```diff
@@ -1,10 +1,20 @@
 from typing import Optional, Sequence, Union
 
+import numpy as np
+import tensorrt as trt
 import torch_tensorrt.dynamo.conversion.impl as impl
 from torch.fx.node import Target
 from torch_tensorrt.dynamo.conversion._ConversionContext import ConversionContext
-from torch_tensorrt.dynamo.conversion.converter_utils import SourceIR, get_trt_tensor
-from torch_tensorrt.fx.converters.converter_utils import set_layer_name
+from torch_tensorrt.dynamo.conversion.converter_utils import (
+    SourceIR,
+    flatten_dims,
+    get_trt_tensor,
+)
+from torch_tensorrt.fx.converters.converter_utils import (
+    Frameworks,
+    set_layer_name,
+    unified_dtype_converter,
```

Comment:
You might need to copy Framework/unified_dtype_converter from torch_tensorrt.fx.converters.converter_utils.
set_layer_name is copied like below.
https://github.com/pytorch/TensorRT/pull/2853/files#diff-f6ddcec33365053481a53cde22972164c5c051197dcd0ec83855b422fb272975R817

Reply:
I've resolve this issue (remove dependency on fx util, move to dynamo)